internal/hud/webview/view.go

```diff
@@ -180,6 +180,8 @@ type View struct {
 
 	TiltCloudUsername   string
 	TiltCloudSchemeHost string
+
+	FatalError string
```

Comment:
i wonder if we should start being better about using omitempty

Reply:
Oh interesting! Why?

Reply:
1) reduce payload size
2) reduce diffs when the snapshot json changes